internal/util/translate.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+var mappings = map[string]string{
+	"FIELD.firstname":    "First name",
+	"FIELD.surname":      "Surname",
+	"FIELD.deputyName":   "Deputy name",
+	"FIELD.telephone":    "Telephone",
+	"FIELD.email":        "Email address",
+	"FIELD.addressLine1": "Address line 1",
+	"FIELD.addressLine2": "Address line 2",
+	"FIELD.addressLine3": "Address line 3",
+	"FIELD.town":         "Town or City",
+	"FIELD.postcode":     "Postcode",
+	"FIELD.county":       "County",
+	"FIELD.country":      "Country",
+}
+
+func Translate(prefix string, s string) string {
```

Comment:
Just to explain this...

We shouldn't really be formatting the timeline events in Sirius and storing them in the database (e.g. having the changeset named "Deputy name"). While this saves time now, it will be a pain if we ever want to change how it is displayed. Instead, we should be translating the placeholders. 

This template function takes a prefix (e.g. "FIELD") and the string to translate (e.g. "firstname") to look up in the translation map. To allow for fall through of unmapped fields, if no translation is found, it just returns the original string. This can then be used just like any other template function, e.g. `{{ translate "FIELD" .FieldName }}`.

In the future, we could use this to do all sorts of translation, e.g. mapping the validation errors.